frontend-next-migration/src/preparedPages/MainPage/ui/_components/sections/PlayWithUs/PlayWithUs.tsx

```diff
@@ -69,6 +69,11 @@ const PlayWithUs = (props: Props) => {
           alt={'Side image with hero'}
           className={cls.sideImg}
         />
+
```

Comment:
I would make this its own shared/ui component for better reusing on other pages as like game site we going to need this also.

you can see how those are build and imported if you check horizontalLines component
<img width="858" alt="image" src="https://github.com/user-attachments/assets/d426c8da-be07-4a13-900f-1db884862da0">

Reply:
Also this component has no translations for multilanguage page so make those with nexts i18n ( there is instructions at wiki)

Reply:
hints: 
![image](https://github.com/user-attachments/assets/341cec04-2772-4e19-aa85-9d77d329e978)
![image](https://github.com/user-attachments/assets/ea1dcf3e-24d2-4e3b-8a26-cd774fcdad6a)
![image](https://github.com/user-attachments/assets/fa33998a-d85f-43e3-96b0-251d4f9731cc)